start-from-zero/sass-architecture/sass/components/_buttons.sass

```diff
@@ -0,0 +1,74 @@
+/*BUTTON FROM THE EVALUATION PAGE + */
+.button-toggle
+    display: inline-block
+    width: 20px
+    height: 20px
+    margin-right: 20px
+    
+
+/*.plus-button 
+    display: inline-block
+    margin: 20px
+    width: 50px
+    height: 50px
+    border: 1px dotted black */
+
+.minus-toggle 
+    display: block
+    width: 20px
+    height: 4px
+    background: $main-color
+    position: relative
+    top: 70%
+    transition: all 0.5s cubic-bezier(.76,-0.26,.15,1.32)
+    transform: translateY(-50%)
+    transform-origin: center center
+
+.minus-toggle::after 
+    content: ""
+    display: block
+    width: 20px
+    height: 4px
+    background: $main-color
+    position: absolute;
+    transition: all 0.5s cubic-bezier(.76,-0.26,.15,1.32)
+    transform-origin: center center
+    transform: rotate(90deg)
+
+.close-2.minus-toggle::after 
+  transform: rotate(0deg)
+  background: transparent
+
+.close-2.minus-toggle 
+  transform: rotate(180deg)
+  top: 45
+
+
+.button-wrap
+    display: flex
+    flex-direction: row
+    justify-content: flex-end
+    
+
+    
+/*BUTTON FROM THE NEW-EVALUATION PAGE*/
+.submit-button
+    background: $main-color

```

Comment:
this class definition is just huge. Also the structure styles and skin styles aren't segregated at all.